index.js

```diff
@@ -432,7 +435,7 @@ module.exports = function (babel) {
         compileTemplate(
           precompile,
           template,
-          state.ensureImport('default', 'ember'),
+          state.ensureImport('createTemplateFactory', '@ember/template-factory'),
```

Comment:
@pzuraq @rwjblue where this module comes from? This causes #345 (e.g. I have Ember.js 3.20.5)

Can't find this module in https://github.com/ember-cli/ember-rfc176-data nor in [ember.js itself](https://github.com/emberjs/ember.js/search?o=asc&q=%22template-factory%22&s=created&type=issues)